translation_py32.py

```diff
@@ -329,15 +329,20 @@ def translateSimple(toTranslate, commands):
     proc_in.stdout.close()
     raise StopIteration(translated.decode('utf-8'))
 
-
+@gen.coroutine
 def translateDoc(fileToTranslate, fmt, modeFile, unknownMarks=False):
     modesdir = os.path.dirname(os.path.dirname(modeFile))
     mode = os.path.splitext(os.path.basename(modeFile))[0]
     if unknownMarks:
         cmd = ['apertium', '-f', fmt,       '-d', modesdir, mode]
     else:
         cmd = ['apertium', '-f', fmt, '-u', '-d', modesdir, mode]
-    proc = Popen(cmd, stdin=fileToTranslate, stdout=PIPE)
+
+    proc_in = yield gen.Task(proc.stdin.write, fileToTranslate)
```

Comment:
TypeError: write() only accepts bytes, unicode, and dict objects

Please test that your code works before you submit for review. If you need help testing, ask on IRC.